submarine-server/server-core/src/test/java/org/apache/submarine/server/rest/workbecnh/CommonDataTest.java

```diff
@@ -29,6 +29,10 @@
 import org.apache.submarine.server.workbench.database.service.SysUserService;
 import org.apache.submarine.server.response.JsonResponse;
 import org.apache.submarine.server.response.JsonResponse.ListResult;
+import org.apache.submarine.server.rest.workbench.SysDeptRestApi;
```

Comment:
@Ray02250418  After you correct the package name, you don't have to import those classes.